lib/widgets/content_list_view/song_tile.dart

```diff
@@ -1,9 +1,12 @@
+import 'dart:math' as math;
+
 import 'package:flutter/material.dart';
 import 'package:sweyer/sweyer.dart';
 
 /// Needed for scrollbar label computations
-const double kSongTileHeight = kSongTileArtSize + _tileVerticalPadding * 2;
 const double _tileVerticalPadding = 8.0;
+double kSongTileHeight(double textScaleFactor) =>
+    (kSongTileArtSize + _tileVerticalPadding * 2) * math.max(0.95, textScaleFactor);
```

Comment:
This distorts the proportions of the art. Scale factor shouldn't ever decrease the size of the tile at all.

Reply:
I think it's better to pass BuildContext instead of double

Reply:
> This distorts the proportions of the art. [...] at some point it surpasses the size of Album and Artist tiles. It also breaks selection indicators.

Hm, I'm unable to replicate any of this. The emulator only allows for a maximum text scale factor of `1.3`, maybe that's not enough.

But I agree, the current formula felt a bit like a hack. Just thinking out loud, the correct solution would be something like
```dart
kSongTileHeight(context) => math.max(kSongTileArtSize, kSongTitleTextSize(context)) + _tileVerticalPadding * 2
```
where `kSongTitleTextSize` would be the height of the song title text with respect to text scaling + the height of the artist name text with respect to the scaling + the padding between the two.

Do you agree?

Reply:
I think dd1c4792a2c3ce934ef292c36faeff627eea6de9 is the correct way to do it. I verified with the layout inspector that we are calculating the exact resulting height of the SongTile.

Reply:
> Hm, I'm unable to replicate any of this. The emulator only allows for a maximum text scale factor of 1.3, maybe that's not enough.

The problem is that even with that value it scales too much unncesessarily. All this empty space is not supposed to be there.

<details>
<summary>screenshot</summary>

![Screenshot_1656788299](https://user-images.githubusercontent.com/39104740/177013157-cff1f14a-35ac-486c-a96f-153bb9ab0631.png)
</details>

Reply:
On my OPPO phone device I'm able to get it up to 1.6

Reply:
> The problem is that even with that value it scales too much unncesessarily. All this empty space is not supposed to be there.

Where is this empty space coming from? We should be scaling with the text, but the padding should remain constant. I'll investigate...

Reply:
Do you still see a problem here with the latest changes? Honestly, I don't see the empty space you are talking about in your screenshot. Could you please clarify where the unnecessary space is?

Reply:
I was talking about the spacing between the title and subtitle, and that it increases. It comes from the text line height scaling. I wanted to lower this a bit, but now looking at this, I think this is fine.